src/Backend/Modules/Analytics/Installer/Uninstaller.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Backend\Modules\Analytics\Installer;
+
+/*
+ * This file is part of Fork CMS.
+ *
+ * For the full copyright and license information, please view the license
+ * file that was distributed with this source code.
+ */
+
+
```

Comment:
I would remove this extra enter

Reply:
Remove this from all installer\uninstaller files?

Reply:
yes, you don't need 2 empty lines, one will suffice